File: backend/src/main/java/com/arhafer/zeldle/dto/Results.java
package com.arhafer.zeldle.dto;

import com.arhafer.zeldle.constant.Result;

public record Results(Result name,
                      Result game,
                      Result gameReleaseDate,
                      Result purpose,
                      Result consumption,
                      Result acquisition,
                      Result range,
                      Result enemyInteraction,
                      Result controlMode) {}

File: frontend/src/types/Results.ts
export type Result = 'CORRECT' | 'INCORRECT' | 'TARGET_HIGHER' | 'TARGET_LOWER' | 'EQUAL'

export interface Results {
    correct: boolean
    name: Result
    game: Result
    gameReleaseDate: Result
    purpose: Result
    consumption: Result
    acquisition: Result
    range: Result
    enemyInteraction: Result
    controlMode: Result
}